nuxeo-features/nuxeo-thumbnail/src/main/java/org/nuxeo/ecm/platform/thumbnail/work/ThumbnailRecomputeWork.java

```diff
@@ -28,7 +28,9 @@
  * Work to recompute the thumbnail of the documents resulting from the provided NXQL query.
  *
  * @since 10.10
+ * @deprecated since 11.1 use {@link RecomputeThumbnailsAction} instead
```

Comment:
can you change deprecation message to be consistent with https://doc.nuxeo.com/corg/java-code-style/#deprecation